python/tvm/micro/contrib/zephyr.py

```diff
@@ -650,10 +650,10 @@ def popen_kwargs(self):
         env = dict(os.environ)
         env["ZEPHYR_BASE"] = self._zephyr_base
 
-        return dict(
+        args = dict(
             args=self._west_cmd
             + [
-                "debug",
+                "attach",
```

Comment:
why this change? I think won't work for non-nrf boards

Reply:
got it, yeah makes sense

Reply:
okay I understand what is happening here. for all openocd runners, you actually need `debug`, because the Zephyr openocd runner doesn't implement `attach`:

```
    def do_run(self, command, **kwargs):
        self.require(self.openocd_cmd[0])
        if ELFFile is None:
            raise RuntimeError(
                'elftools missing; please "pip3 install elftools"')

        self.cfg_cmd = []
        if self.openocd_config is not None:
            self.cfg_cmd = ['-f', self.openocd_config]

        if command == 'flash' and self.use_elf:
            self.do_flash_elf(**kwargs)
        elif command == 'flash':
            self.do_flash(**kwargs)
        elif command == 'debug':
            self.do_debug(**kwargs)
        elif command == 'load':
            self.do_load(**kwargs)
        else:
            self.do_debugserver(**kwargs)
```

it looks like this limitation is not addressed in zehpyr 2.5.0 either. so, wondering if we need this change after all?